### PR TITLE
feat(securityhub): add L2 Hub construct

### PR DESCRIPTION
**Pull request by thomasklinger1234**
----

Initial commit to support SecurityHub by introducing the `Hub` construct. In addition, it also adds support for CloudWatch Events.

Checklist

- [X] PR title
- [X] PR description
- [X] Testing: Unit tests are added
- [X] Documentation: README for module has been updated

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/@aws-cdk/aws-securityhub/lib/hub.ts [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+/**
+ * Properties for SecurityHub Hub.
+ */
+export interface HubProps {
+    /**
+     * A key-value map of resource tags of this Hub.
+     *
+     * @default - Empty tags
+     */
+    readonly tags?: { [key: string]: string };
```
Comment (rix0rrr) [inline packages/@aws-cdk/aws-securityhub/lib/hub.ts]: Do tags not work out-of-the box using the aspect mechanism?

File: packages/@aws-cdk/aws-securityhub/package.json [hunk 2]
```diff
@@ -62,17 +62,6 @@
     "url": "https://aws.amazon.com",
     "organization": true
   },
-  "jest": {
```
Comment (rix0rrr) [inline packages/@aws-cdk/aws-securityhub/package.json]: Why are you removing this? I would prefer 'jest' tests to nodeunit tests, if we are starting fresh anyway.

File: packages/@aws-cdk/aws-securityhub/lib/hub.ts (hunk 1 above)
Comment (thomasklinger1234) [reply packages/@aws-cdk/aws-securityhub/lib/hub.ts]: Yes, I think you are right.

File: packages/@aws-cdk/aws-securityhub/package.json (hunk 2 above)
Comment (thomasklinger1234) [reply packages/@aws-cdk/aws-securityhub/package.json]: I have enabled it again.

**Comment by rix0rrr** [comment]
> I have enabled it again.

Sure and thanks, but that also implies that the tests should be rewritten as Jest tests. It's probably just the filename, rename to `*.test.ts`.

Build fails now with:

```
@aws-cdk/aws-securityhub: Error: Jest is enabled, but 1 nodeunit tests were found!
```

**Pull request closed without merging**